nss-day-cohort-30/bangazon-site-plucky-pikes
# More than one order report

# Description
Multiple Orders report is now an options on the reports drop-down. The report shows users who have two or more open orders.

## Type of change
[X] New feature (non-breaking change which adds functionality)

# Testing Instructions for Change Made
1. `git fetch --all`
1. `git checkout more-than-one-order-report`
1. go to Bangazon site and log in
1. click reports in the navigation bar and select multiple orders
1. verify that the First Name, Last Name, and number of orders appear for users with multiple open orders.
1. verify that any user with fewer than two open orders does not appear
1. self high-five


# Checklist:
